### ISSUE TITLE
Atom fails to reopen windows when relaunching

### ISSUE THREAD

**Issue by alexchandel** [atom/atom#10111]
1. Open Atom 1.3.2 on OS X 10.11.1.
2. Add a project folder, leave some tabs open, and close Atom.
  * Repeat Step 2 multiple times for extra fail
3. Reopen Atom by double-clicking a file associated with it, outside any project.
4. Atom completely forgets what was last open, and launches with A window open to double-clicked file's folder, with single tab for the double-clicked file.

After Step 4, depending on the circumstances, Atom may recover A open project (for example by `atom project/folder`. However, double-clicking a file within that folder, or manually reopening a file, usually results in the open tabs being lost, and multiple open windows are always lost.

**Comment by 50Wliu** [comment]
@alexchandel Can this be reproduced in safe mode?  The 1.4.0 betas?

**Comment by snario** [comment]
Same as #10044. Has never worked for me

**Comment by lee-dohm** [comment]
I agree that this is a duplicate of #10044. Please subscribe there for updates.
